src/commands/admin/prune.ts

```diff
@@ -0,0 +1,107 @@
+import { Command } from '@lib/types/Command';
+import { ROLES } from '@root/config';
+import { adminPerms } from '@root/src/lib/permissions';
+import { GuildMember, Message, MessageActionRow, MessageButton, MessageEmbed } from 'discord.js';
+
+const PRUNE_TIMEOUT = 30;
+
+export default class extends Command {
+
+	description = `Prunes all members who don't have the <@&${ROLES.VERIFIED}> role`;
+	runInDM = false;
+
+	permissions(msg: Message): boolean {
```

Comment:
Might this be better off using the bot master perms? Looking at permissions.ts I assume this includes all members of the discord team. It would be good for futureproofing or in case someone (somehow!) wreaks havoc.